server/cabal.project.freeze

```diff
@@ -100,255 +101,266 @@ constraints: any.Cabal ==2.4.0.1,
-             any.hedgehog ==0.6.1,
+             any.hashtables ==1.2.3.4,
+             hashtables -bounds-checking -debug -detailed-profiling -portable -sse42 +unsafe-tricks,
+             any.haskell-lexer ==1.1,
+             any.hasql ==1.4.2,
+             any.hasql-pool ==0.5.1,
+             any.hasql-transaction ==1.0.0.1,
+             any.hedgehog ==1.0.2,
              any.hourglass ==0.2.12,
-             any.hsc2hs ==0.68.6,
+             any.hsc2hs ==0.68.7,
              hsc2hs -in-ghc-tree,
-             any.hspec ==2.6.1,
-             any.hspec-core ==2.6.1,
-             any.hspec-discover ==2.6.1,
+             any.hspec ==2.7.1,
+             any.hspec-core ==2.7.1,
+             any.hspec-discover ==2.7.1,
              any.hspec-expectations ==0.8.2,
              any.hspec-expectations-lifted ==0.10.0,
-             any.http-api-data ==0.4,
+             any.http-api-data ==0.4.1.1,
              http-api-data -use-text-show,
-             any.http-client ==0.5.14,
+             any.http-client ==0.6.4.1,
              http-client +network-uri,
              any.http-client-tls ==0.3.5.3,
              any.http-date ==0.0.8,
              any.http-types ==0.12.3,
-             any.http2 ==1.6.5,
+             any.http2 ==2.0.4,
              http2 -devel,
              any.hvect ==0.4.0.0,
              any.immortal ==0.2.2.1,
-             any.insert-ordered-containers ==0.2.1.0,
-             any.integer-gmp ==1.0.2.0,
+             any.indexed-profunctors ==0.1,
+             any.insert-ordered-containers ==0.2.3.1,
+             any.integer-gmp ==1.0.3.0,
              any.integer-logarithms ==1.0.3,
              integer-logarithms -check-bounds +integer-gmp,
              any.invariant ==0.5.3,
-             any.iproute ==1.7.7,
-             any.jose ==0.8.0.0,
+             any.iproute ==1.7.9,
+             any.jose ==0.8.2.0,
+             jose -demos,
              any.js-flot ==0.8.3,
              any.js-jquery ==3.3.1,
              any.kan-extensions ==5.2,
-             any.keys ==3.12.2,
-             any.lens ==4.17.1,
+             any.lens ==4.19.2,
              lens -benchmark-uniplate -dump-splices +inlining -j -old-inline-pragmas -safe +test-doctests +test-hunit +test-properties +test-templates +trustworthy,
-             any.lens-aeson ==1.0.2,
+             any.lens-aeson ==1.1,
              lens-aeson +test-doctests,
-             any.libyaml ==0.1.1.0,
+             any.libyaml ==0.1.2,
              libyaml -no-unicode -system-libyaml,
-             any.lifted-async ==0.10.0.4,
+             any.lifted-async ==0.10.0.6,
              any.lifted-base ==0.2.3.12,
-             any.list-t ==1.0.3.1,
+             any.list-t ==1.0.4,
              any.loch-th ==0.2.2,
-             any.math-functions ==0.3.1.0,
-             math-functions -system-expm1,
-             any.memory ==0.14.18,
+             any.math-functions ==0.3.3.0,
+             math-functions +system-erf +system-expm1,
```

Comment:
Out of interest, why is are these flags on `math-functions` relevant for us? Do we get a compilation error in another dependency otherwise? Surely we're not actually using these functions.